pkg/plugin/state.go

```diff
@@ -964,6 +964,8 @@ func (p *AutoscaleEnforcer) readClusterState(ctx context.Context, logger *zap.Lo
 		} else if pod.Spec.NodeName == "" {
 			logSkip("VM pod Spec.NodeName = \"\" (maybe it hasn't been scheduled yet?)")
 			continue
+		} else if util.PodCompleted(pod) {
```

Comment:
strictly speaking you don't need any of these because there's no way past any of these if statements (there used to be one of the really annoying linters that people don't use any more that chided you for the unnecessary else, and while the linter is turned off it's still in my heart.)

Reply:
Yeah. I often will include "unnecessary" control flow like that because the "else" is implied _anyways_ if the block above diverges (returns / continues / etc), and I tend to feel that writing the "else" explicitly makes it a bit more clear. But... :shrug: